src/styles/blocks/header.scss

```diff
@@ -0,0 +1,57 @@
+.header {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  line-height: 14px;
+  padding: 0 50px;
+}
+
+.logo {
+  height: 40px;
+  width: 40px;
+}
+
+.nav {
+
+  &__list {
+    display: flex;
+    list-style: none;
+    padding-left: 0;
+    margin: 0;
+  }
+  
+  &__link {
+    color: $major-text-color;
+    text-decoration: none;
+    display: flex;
+    padding: 23px 0;
+    text-transform: uppercase;
+    font-size: 12px;
+    font-weight: 500;
+
+    &:hover {
+      color: $light-blue-color;
+    }
+  }
+  
+  &__item {
+    margin-left: 20px;
+  }
```

Comment:
```suggestion
  &__item:not(:first-child) {
    margin-left: 20px;
  }
```

Reply:
it is can fix this problem
![Screenshot from 2023-03-02 21-09-12](https://user-images.githubusercontent.com/33029864/222527838-e3ced3f9-29c4-4ad4-be8a-475603c1542c.png)